.eslintignore

```diff
@@ -1 +1,2 @@
 dist/*
+karma.conf.cjs
```

Comment:
I wouldn't ignore this, but get rid of the no dynamic imports rule instead

Reply:
Thought there must be a reason it was disabled and now it will still not allow it in the code.

Don't know the drawbacks of dynamic imports it so I can get rid of the rule